src/handbook/customer/marketing/messaging.md

```diff
@@ -1,29 +1,52 @@
 ---
-navTitle: Boiler Plate Descriptions
-navGroup: Sales & Marketing
+navTitle: FlowFuse Messaging
 ---
 
-# Marketing Boilerplate
+# {{ navTitle }}
 
-## Tagline
+
+## Audience
+
+FlowFuse is a business to business (B2B) company in the industrial markets to
+innovate with software through low-code development. The end user is a skilled
+worker, but not always skilled in automation through software.
+
+FlowFuse often sells to IT teams that want to enable the OT teams to participate
+in digital solution building.
+
+FlowFuse focusses on the engineer for messaging, this is due to the adoption of
+our software being grass-roots.
+
+### Ideal Customer Profile (ICP)
+
+FlowFuse caters to a brought range of customers, many of which do not fall into
```

Comment:
```suggestion
FlowFuse caters to a broad range of customers, many of which do not fall into
```

Reply:
Should we say "appeals" to a broad range of customers, in which case not everyoone falling into our ICP makes sense.